tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -31,130 +30,107 @@ public void Constructor_NullRepository_Throws()
         [Test]
         public async Task GetAllAsync_NoOrdersExist_ReturnsEmptyResult()
         {
-            using var controller = OrdersControllerBuilder
-                .Create()
-                .WithListOrders(new List<Order>())
-                .Build();
-
-            var result = await controller.GetAllAsync();
+            var context = OrdersControllerTestContext.Setup();
 
-            result.Should().BeOfType<OkObjectResult>();
-            result.Should().NotBeNull();
+            using var controller = context.OrdersController;
 
-            var objectResult = result as OkObjectResult;
-            objectResult.Value.Should().BeOfType<List<OrderModel>>();
-
-            var ordersResult = objectResult.Value as List<OrderModel>;
-            ordersResult.Count().Should().Be(0);
+            var result = await controller.GetAllAsync(Guid.Empty) as OkObjectResult;
+            var orders = result.Value as List<OrderModel>;
+            orders.Count.Should().Be(0);

```

Comment:
I'd consider using Fluent Assertion's method for asserting whether a collection is empty. 

Do you think it would look better with
`orders.Should().BeEmpty();`?

Reply:
👍